plugins/disk/quota2percent_

```diff
@@ -1,143 +1,205 @@
 #!/bin/bash
+# -*- sh -*-
+: <<=cut
+
+=head1 NAME
+
+quota2percent - Plugin to show disk usage in percent of quota hard limit.
+
+=head1 APPLICABLE SYSTEMS
+
+All systems with "bash", "quota", "repquota" and "munin"
```

Comment:
in addition to the technical requisites, I would suggest something like "Systems with multiple users and individual storage space limitations administered via 'quota'".